lib/node_modules/@stdlib/blas/ext/base/ssumpw/src/main.c

```diff
@@ -37,9 +37,9 @@
 * @param strideX  stride length
```

Comment:
@headlessNode Don't we also need to remove the `is_nanf` dependency both here and in the `manifest.json` file?